### PR TITLE
feat: add `subscribed_size` method on subject

### PR DESCRIPTION
**Pull request by M-Adoo**

### CODE REVIEW

**Comment by M-Adoo** [comment]
@mergifyio rebase

File: src/subject/local_subject.rs [hunk 1]
```diff
@@ -25,6 +25,11 @@ impl<'a, Item, Err> LocalObservable<'a> for LocalSubject<'a, Item, Err> {
   }
 }
 
+impl<'a, Item, Err> LocalSubject<'a, Item, Err> {
+  pub fn subscribed_size(&mut self) -> usize {
+    self.observers.borrow_mut().len()
```
Comment (Mnwa) [inline src/subject/local_subject.rs]: Why there is a used _mut method?

Comment (M-Adoo) [reply src/subject/local_subject.rs]: fixed it, please review continue.

**Comment by M-Adoo** [comment]
@mergifyio rebase

**Comment by M-Adoo** [comment]
@Mergifyio rebase

**Pull request merged**